http/README.md

```diff
@@ -35,10 +35,36 @@ actions {
         port = 3000
         allowedRequestHeaders = ["Content-Type"]
       }
+      responseOptions {
+        predicates = [JSON]
+        forceJson = false
+      }       
     }
   }
 }
 ```
 
+- `endpointOptions` config describes quite basic request parameters as `path`, `domain`, `port` and `allowedRequestHeaders` which
+are necessary to perform http request 
+- `responseOptions` config is responsible for handling incoming request properly. Here we can specify `predicates` - it's array
+containing Vert.x response predicates, to get more familiar with it, please visit [this page](https://vertx.io/blog/http-response-validation-with-the-vert-x-web-client/).
+Providing `JSON` predicate causes `Content-Type` check - when `Content-Type` won't be equal to `application/json` we'll get
+error transition. We can also specify `forceJson` param. When `Content-Type` won't be equal to `application/json` and `forceJson`
+is true, response will be processed as json. If it won't be json, request ends with error transition.
+
+Table below shows default behaviour of HttpAction depending on provided `responseOptions` config and response:
+
+| Content-Type     | forceJSON | JSON predicate | Body | Transition | Response |
+| ---------------- |:---------:| --------------:|  ---:|  ---------:| --------:|
+| application/json | false     | -              | JSON | _success   | JSON     |
+| application/text | true      | -              | JSON | _success   | JSON     |
+| application/json | false     | JSON           | JSON | _success   | JSON     |
+| application/text | false     | -              | JSON | _success   | text     |
+| application/json | false     | -              | RAW  | _error     | -        |
+| application/text | true      | -              | RAW  | _error     | -        |
+| application/json | false     | JSON           | RAW  | _error     | -        |
+| application/text | false     | JSON           | JSON | _error     | -        |
+| application/text | true      | JSON           | JSON | _error     | -        |
```

Comment:
What about `| application/text | false | - | RAW |` ?
I mean, there is no content-type set (or it is e.g. text), we do not force json and no predicate is provided?
If I understand correctly we will also end up with `| _success   | text     |`?